src/api/models/mermaid.py

```diff
@@ -1738,3 +1738,24 @@ class AuditRecord(JSONMixin):
     event_by = models.ForeignKey("Profile", on_delete=models.SET_NULL, null=True, blank=True)
     model = models.CharField(max_length=100)
     record_id = models.UUIDField(db_index=True)
+
+
+class Notification(BaseModel):
```

Comment:
So `created_on` in `BaseModel` would be used for the datestamp, right?

Reply:
yeah